app/modules/asset_groups/models.py

```diff
@@ -70,6 +70,7 @@ class AssetGroupSighting(db.Model, HoustonModel):
 
     # May have multiple jobs outstanding, store as Json obj uuid_str is key, In_progress Bool is value
     jobs = db.Column(db.JSON, default=lambda: {}, nullable=True)
+    detection_start = db.Column(db.DateTime, index=True, default=None, nullable=True)
```

Comment:
i wonder if this can (or _should_?) be calculated off of `progress_detection` instead?   this might be a question for @bluemellophone ... i am not sure of the subtleties of how/when `progress_detection` is created (and thus possibly could use `progress_detection.created` ... etc.

Reply:
I'm inclined to use a dedicated column for now until the progress API is a bit more fleshed out and tested.  At the moment, it's ok that this value is duplicated in two places... but with the acknowledgment that the progress API is here to stay and things like this will eventually need to be consolidated / deprecated.